libraries/SITL/SIM_QuadPlane.cpp

```diff
@@ -126,6 +126,7 @@ void QuadPlane::update(const struct sitl_input &input)
     accel_body += quad_accel_body;
 
     update_dynamics(rot_accel);
+    update_external_payload(input);
```

Comment:
I added this so I can get precland working in SITL, it also brings over other SITL instances - I haven't tested them all yet. Is there a reason this wasn't included before?

Reply:
that's fine